src/lib/containers/QueryWrapper.tsx

```diff
@@ -458,7 +478,7 @@ export default class QueryWrapper extends React.Component<
     }
     const loadingCusrorClass = isLoading ? 'SRC-logo-cursor' : ''
     return (
-      <div className={`SRC-wrapper ${loadingCusrorClass}`}>
+      <div className={`SRC-wrapper ${loadingCusrorClass} ${isFaceted ? 'has-facets' : ''}`}>
```

Comment:
Layout changes depending on if facet filters are being shown (reclaims facet filter panel section for the table)